cmd/kyverno/main.go

```diff
@@ -78,6 +78,7 @@ func main() {
 		go http.ListenAndServe("localhost:6060", nil)
 	}
 
+	// TODO: Do we need to print here? It anyways prints empty values
```

Comment:
Why is it empty? If using the Makefile to build the binary, it should have version info built.

Reply:
It is empty when we are running in debug mode. 

I have removed this comment.